src/shadowbox/docker/Dockerfile

```diff
@@ -16,7 +16,7 @@
 FROM node:8.15.0-alpine
 
 # Versions can be found at https://github.com/Jigsaw-Code/outline-ss-server/releases
-ARG SS_VERSION=1.0.8
+ARG SS_VERSION
```

Comment:
Unrelated to your change: Would you mind placing all the ARGS together and all the LABELs together? I think it will clarify what the needed args are, and what the full set of labels are.